### ISSUE TITLE
Container exists instantly after startup

### ISSUE THREAD

**Issue by wis-niowy** [testcontainers/sshd-docker#5]
Hi, I'm trying to run a test container based on `testcontainers/sshd:1.1.0 image`.

Here is my code that attempts to do so:
```c#
var container = new ContainerBuilder()
            .WithImage("testcontainers/sshd:1.1.0")
            .WithPortBinding(22, true)
            .WithExposedPort(22)
            .WithDockerEndpoint("tcp://localhost:2375")
            .Build();

await container.StartAsync();
```

When running the code I lookup the `Docker Desktop` - and I can see that the container gets created, starts running and then instantly exits (with code 0).
What looks odd is that port mapping actually is `0:22` - so it apparently failed to assign a random port of my local machine.
When I select a port for the mapping, e.g. `.WithPortBinding(55548, 22)`, the mapping in the Docker Desktop looks fine (`55548:22`) - but the container still exits instantly.

The exception that I get in the code is: `System.TimeoutException : The operation has timed out.`

Everything is fine when I use `testcontainers/helloworld:1.1.0` image, with the following code:
```c#
var container = new ContainerBuilder()
            .WithImage("testcontainers/helloworld:1.1.0")
            .WithPortBinding(8080, true)
            .WithExposedPort(8080)
            .WithDockerEndpoint("tcp://localhost:2375")
            .Build();

await container.StartAsync();
```
The container stands up and runs properly - and I can successfully hit it with http request (via `HttpClinet()`).

So I am wondering if I am missing something or there is an issue with this image.
`testcontainers/helloworld` does not meet my requirements as I need an instance with a sshd running to test my ssh/sftp client locally.

My OS is Windows 10.

**Comment by theshubhamp** [comment]
Happens because the Docker image is not setup with an entrypoint that starts up SSHD.

Permitting root login and configuring the demon to not detach gets it working:
```
docker run -it -p 9022:22 testcontainers/sshd sh -c 'echo "PermitRootLogin yes" >> /etc/ssh/sshd_config; /usr/sbin/sshd -D'
```

Is this repo / image still maintained ? If so, I can send in a PR. Otherwise can be work around on ContainerBuilder directly.

**Comment by mdelapenya** [comment]
This image is used by the testcontainers libraries to provide an SSH tunnel, exposing host ports to a container. We fixed this in #7, so switching to `testcontainers/ssh:1.2.0` would work out-of-the-box for you.

For that reason I'm closing this issue. Thanks!
